### ISSUE TITLE
Code is full of bugs

### ISSUE THREAD

**Issue by Julesaiyy** [hust-linyi/CAT-Net#4]
Thank you very much for your wonderful work. I have many bugs when I run the code, such as Res101Encoder has no output_dim parameter and there is no high_avg_pool function in the model, etc., so I would like to ask the author to double check the code.

**Comment by CYF2000127** [comment]
Thanks for your attention!  We are very sorry, we will check the code soon.

**Comment by WangBingJian233** [comment]
> 感谢您的关注！我们非常抱歉，我们会尽快检查代码。

Hello, there is still a problem with your code. For example fewshot.py line 115, #qry_fts = torch.cat([qry_fts_reshaped, corr_query_mask], dim=2) # N x B x (C + 1) x H' x W'. Here, this is one of the core of the model, but you annotated it. After removing the annotations, it was modified and run as described in the paper, and the loss gradient descent was obviously wrong. Please update the error code again.

**Comment by CYF2000127** [comment]
> Thank you very much for your wonderful work. I have many bugs when I run the code, such as Res101Encoder has no output_dim parameter and there is no high_avg_pool function in the model, etc., so I would like to ask the author to double check the code.

We have fixed this bug, you can check now! Thanks!
